site/en/tutorials/images/transfer_learning.ipynb

```diff
@@ -683,7 +683,9 @@
       "source": [
         "### Train the model\n",
         "\n",
-        "After training for 10 epochs, you should see ~94% accuracy on the validation set.\n"
+        "After training for 10 epochs, you should see ~94% accuracy on the validation set.\n",
```

Comment:
This change may have introduced 1 of 2 issues that broke the notebook (usually, it's because of the end of line `\n`s in JSON). I will fix this in a separate commit by reverting this change. Please edit using Colab/Jupyter in future.